docker-compose.yml

```diff
@@ -96,7 +96,7 @@ nginx:
   environment:
     - CONSUL_AGENT=1
   labels:
-    - triton.cns.services=nginx
+    - triton.cns.services=wp-nginx,nginx
```

Comment:
Though I chose to make a breaking change elsewhere, I'm preserving the un-namespaced nginx here because I can't change my Bitly link